portfolio/src/main/webapp/index.html

```diff
@@ -6,9 +6,17 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getjson()">
-    <div id="content">
+  <body onload="getjson(); getLoginStatus();">
+    <form>
```

Comment:
nit: I don't think you need the `form` element here, though it'll be harmless anyway. Try to remember removing this later.